admin/config.yml

```diff
@@ -110,40 +110,89 @@ collections:
         collapsed: true
         required: false
         fields:
-            - { label: 'Column 1', name: 'col1', widget: 'markdown', required: false, i18n: true }
-            - { label: 'Banner Text', name: bannertext, widget: markdown, required: false, i18n: true}
-            - { label: 'Text Banner Column 1', name: bannercol1, widget: markdown, required: false, i18n: true }
-            - { label: 'Text Banner Column 2', name: bannercol2, widget: markdown, required: false, i18n: true }
-            - { label: 'Column 2', name: 'col2', widget: 'markdown', required: false, i18n: true }
+          - { label: 'Column 1', name: 'col1', widget: 'markdown', required: false, i18n: true }
```

Comment:
This is Prettier auto-formatting. Shouldn't have any effect and would want to standardize on this format anyways.